packages/manager/apps/dedicated/client/app/ip/components/list/byoip/slice/ip-ip-byoip-slice.controller.js

```diff
@@ -92,6 +92,13 @@ export default class IpByoipSliceController {
       this.errorMessage = this.$translate.instant(
         'ip_byoip_aggregate_ip_doesnt_exist_error_message',
       );
+    } else if (
+      error.status === 400 &&
+      /routed to a service.*park it first/.test(error.data.message)
```

Comment:
If possible to move this regex in constant file

Reply:
Done Guillaume